lib/dash/content_protection.js

```diff
@@ -92,22 +92,22 @@ shaka.dash.ContentProtection.Element;
  * @private
  */
 shaka.dash.ContentProtection.defaultKeySystems_ = new Map()
-    .set('urn:uuid:1077efec-c0b2-4d02-ace3-3c1e52e2fb4b',
-         'org.w3.clearkey')
-    .set('urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed',
-         'com.widevine.alpha')
-    .set('urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95',
-         'com.microsoft.playready')
-    .set('urn:uuid:f239e769-efa3-4850-9c16-a903c6932efb',
-         'com.adobe.primetime');
+  .set('urn:uuid:1077efec-c0b2-4d02-ace3-3c1e52e2fb4b',
+    'org.w3.clearkey')
+  .set('urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed',
+    'com.widevine.alpha')
+  .set('urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95',
+    'com.microsoft.playready')
+  .set('urn:uuid:f239e769-efa3-4850-9c16-a903c6932efb',
+    'com.adobe.primetime');
 
 
 /**
  * @const {string}
  * @private
  */
 shaka.dash.ContentProtection.MP4Protection_ =
-    'urn:mpeg:dash:mp4protection:2011';
+  'urn:mpeg:dash:mp4protection:2011';
```

Comment:
Same comment as before, style guide violations. Please update the change to follow the style guide. If you run `./build/all.py` it should (but not always) help point out violations.

However a lot of these problems are on lines that you only changed the formatting of. Am I missing something?

Reply:
No, you're not missing anything.  I did 'Format Document' in my editor and it changed all those lines.  I will go back and revert them.  FWIW, I did `./build/all.py` before submitting this PR and it came back with no issues.